src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -79,7 +79,7 @@ function NewContactMethodPage(props) {
     const submitForm = useCallback(() => {
         const phoneLogin = LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(login));
         const parsedPhoneNumber = parsePhoneNumber(phoneLogin);
-        const userLogin = parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : login;
+        const userLogin = parsedPhoneNumber.possible ? `${parsedPhoneNumber.number.e164}${CONST.SMS.DOMAIN}` : login;
```

Comment:
NAB: Maybe a comment in these cases could be helpful otherwise one wouldn't understand the significance of appending the domain here.

Reply:
It seems we ended up fixing this twice at the same time. Either way, should leave both fixes? https://github.com/Expensify/App/pull/18473 Even if they are redundant, it will make it more bullet proof? 😄

Reply:
Not exactly bullet proof :sweat_smile: I have commented on the linked PR https://github.com/Expensify/App/pull/18473#issuecomment-1540506140.